File: C_Programs/floatarray.cpp
// wap to create a float array 0 10 elements to store salary of 10 employees
#include<stdio.h>
int main()
{
	int cnt,num;
	double salary[10];
	for(cnt=0;cnt<10;cnt++)
	{
		printf("enter salary of %d : ",cnt);
		scanf("%lf",&salary[cnt]);
}
for(cnt=0;cnt<10;cnt++)
{
	printf("\n salary of %d = %.2lf",cnt,salary[cnt]);
}
}

File: Basics_Java_Programs/FloatArray.java
//wap to create a float array of 5 elements and accept the elements from the user.
import java.util.*;
public class FloatArray {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        float num[] = new float[5];
        for (int i = 0; i < 5; i++) {
            System.out.println("enter number");
            num[i] = scan.nextFloat();
        }
        for (int i = 0; i < 5; i++) {
            System.out.println("number is " + num[i]);
        }
    }
}